src/app/static/src/tests/components/projects/projectHistory.test.ts

```diff
@@ -463,14 +472,162 @@ describe("Project history component", () => {
         expect(mockPromoteVersion.mock.calls[0][1]).toStrictEqual(
             {
                 "name": "newProject",
-                "note": "",
+                "note": "version notes",
                 "version": {
                     "projectId": 1,
                     "versionId": "s11"
                 }
             });
     });
 
+    it("invokes versionNote action when ok button is triggered", async () => {
+        const wrapper = getWrapper(testProjects);
+        const copyLink = wrapper.find("#v-s11").find(".version-cell.edit-cell").find("button");
+        copyLink.trigger("click");
+        await Vue.nextTick();
+
+        const modal = wrapper.findAll(".modal").at(3);
+        const textarea = modal.find("textarea");
+        textarea.setValue("new notes");
+
+        const noteText = textarea.element as HTMLTextAreaElement
+        expect(noteText.value).toBe("new notes")
+        expect(wrapper.vm.$data.editedNote).toBe("new notes")
+
+        const okBtn = modal.find(".modal-footer").findAll("button").at(0);
+        await okBtn.trigger("click");
+
+        expect(mockUpdateVersion.mock.calls.length).toBe(1);
+        expect(mockUpdateVersion.mock.calls[0][1]).toStrictEqual(
```

Comment:
Maybe check that mockUpdateProject *hasn't* been called? And the opposite for the below test of updating project note. 
Should maybe also check here that the dialog is hidden, as you do for the cancel tests.